kindergarten-garden: split description into introduction and instructions (#2439)
exercism/problem-specifications: exercises/kindergarten-garden/introduction.md

# Introduction

The kindergarten class is learning about growing plants.

> The teacher thought it would be a good idea to give them actual seeds, plant them in actual dirt, and grow actual plants.

Reviewer: Grammar nitpick: apparently the class is planted.

```suggestion
The teacher thought it would be a good idea to have them take actual seeds, plant them in actual dirt, and grow actual plants.
```

Reviewer 2: ```suggestion
The teacher thought it would be a good idea to give the class seeds to plant and grow in the dirt.
```

To this end, the children have put little cups along the window sills and planted one type of plant in each cup.

> The children chose randomly from four available types of seeds: grass, clover, radishes, and violets.

Reviewer 3: ```suggestion
The children got to pick their favorites from four available types of seeds: grass, clover, radishes, and violets.
```
